scripts/build_anaconda.sh

```diff
@@ -339,6 +339,10 @@ if [[ -n $cuda_ver ]]; then
   #caffe2_cmake_args+=("-DUSE_NCCL=OFF")
   #caffe2_cmake_args+=("-DUSE_GLOO=OFF")
   #caffe2_cmake_args+=("-DCAFFE2_STATIC_LINK_CUDA=ON")
+
+  if [[ $upload_to_conda ]]; then
+    caffe2_cmake_args+=("-DCUDA_ARCH_NAME=All")
```

Comment:
All usually builds for things like 2.0 and 2.1 and 3.0, which no one even uses (and CUDA9 dont support 2.x).